src/moscot/plotting/_utils.py

```diff
@@ -489,12 +491,17 @@ def _plot_temporal(
                 title=titles[i],
                 size=size,
                 ax=ax,
-                show=show,
+                show=False,
+                return_fig=False,
                 **kwargs,
             )
     if suptitle is not None:
+        if fig is None:
```

Comment:
I think we can extract the figure from the raveled axes, no?
Otherwise, I think error is too strong, would just warn.

Reply:
No, I don't think so. Fine, will do a warning